demo-app/src/LocationsMap.jsx

```diff
@@ -19,7 +27,14 @@ function LocationsMap() {
   return (
     <Box ref={containerRef} flex background="background-contrast">
       {geolocation && (
-        <Map id="map" ref={mapContainerRef} center={geolocation} zoom={6}>
+        <Map
+          id="map"
+          ref={mapContainerRef}
+          center={LocationBounds({ locations: geojsonLocations }).getCenter()}
+          zoom={6}
+          zoomControl={false}
+        >
+          <Controls locations={geojsonLocations} />
```

Comment:
Having this fit the geoJSON bounds makes it difficult to assess the capability here.

Reply:
Perhaps we eliminate the GeoJSON marker cluster and focus on the generated locations?

We may need to move the generateLocations call into a useEffect and store locations in state...

Reply:
Sounds good, let me have a look into this